### ISSUE TITLE
Blog teasers don't parse out YouTube video embeds correctly, looks bad

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/jeffgeerling-com#115]
See: https://www.jeffgeerling.com/blog

<img width="1280" alt="Screen Shot 2020-05-29 at 7 10 30 PM" src="https://user-images.githubusercontent.com/481677/83314462-209b0d00-a1e0-11ea-9556-84a030763f0f.png">

@mglaman found this, so credit to him... if you look at the blog page and one of the entries has a YouTube embed, it's all garbled. It should either display the embed, or drop it, not display a bunch of random CSS!

**Comment by geerlingguy** [comment]
@mglaman Yes, that's exactly what's happening. I grab the embed code from http://embedresponsively.com — which tosses it into `<style>` tags. I could maybe put HTML comments around that?

**Comment by mglaman** [comment]
Ah, so it is intended to be embedded as part of the blog content. I think the only workaround is to add that kind of CSS to your site's theme, or manually set the teaser to _not_ include the video.

**Comment by geerlingguy** [comment]
Closing as I'm migrating to Hugo: https://github.com/geerlingguy/jeffgeerling-com/issues/158
